EasyRestProjectNetTeam2/EasyRestPages/ManageRestaurantsPage.cs

```diff
@@ -0,0 +1,90 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EasyRestProjectNetTeam2.EasyRestPages
+{
+    public class ManageRestaurantsPage : BasePage
+    {
+        public ManageRestaurantsPage(IWebDriver driver) : base(driver) { }
+
+        private const int TimeToWait = 20;
+        [FindsBy(How = How.XPath, Using = "//span[(text()= 'details')]/ancestor::a")]
```

Comment:
please add one empty line between the field so that it would be better readability

Reply:
Fixed